### DISCUSSION TITLE
How to show the X and Y axials in the Pole figure?

### DISCUSSION THREAD

**Discussion by louis96108** [mtex-toolbox/mtex#1772] [Ask Anything]
How to show the X and Y axials in the Pole figure?

**Comment by lfgmorales** [comment]
<https://mtex-toolbox.github.io/Annotations.html>
Annotations | MTEX<https://mtex-toolbox.github.io/Annotations.html>
mtex-toolbox.github.io<https://mtex-toolbox.github.io/Annotations.html>
        [favicon.png] <https://mtex-toolbox.github.io/Annotations.html>



Luiz F. G. Morales
ScopeM - ETH Zürich
Sent from iPhone

On 18 Mar 2023, at 06:17, louis96108 ***@***.***> wrote:

﻿

How to show the X and Y axials in the Pole figure?

—
Reply to this email directly, view it on GitHub<https://github.com/mtex-toolbox/mtex/discussions/1772>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALE5IN3XAQVUYW3DXHMENVLW4VANNANCNFSM6AAAAAAV7IGOEY>.
You are receiving this because you are subscribed to this thread.Message ID: ***@***.***>

[ { ***@***.***": "http://schema.org", ***@***.***": "EmailMessage", "potentialAction": { ***@***.***": "ViewAction", "target": "https://github.com/mtex-toolbox/mtex/discussions/1772", "url": "https://github.com/mtex-toolbox/mtex/discussions/1772", "name": "View Discussion" }, "description": "View this Discussion on GitHub", "publisher": { ***@***.***": "Organization", "name": "GitHub", "url": "https://github.com" } } ]

**Comment by louis96108** [comment]
Hi,thanks for your reply. I wanna get not only the labels but also the axial line.

**Reply by zmichels** [reply]
You can use:
plot([vector3d.X, vector3d.Y], 'plane')
